modules/abstract-eth/src/abstractEthLikeNewCoins.ts

```diff
@@ -2474,4 +2477,38 @@ export abstract class AbstractEthLikeNewCoins extends AbstractEthLikeCoin {
     }
     return Buffer.concat(parts);
   }
+
+  /**
+   * Build the data to transfer an ERC-721 or ERC-1155 token to another address
+   * @param params
+   */
+  buildNftTransferData(params: NFTTransferOptions & { fromAddress: string }): string {
```

Comment:
I see that this is only implemented for `abstractEthLikeNewCoins` and not `abstractEthLike`. That's intentional right? I'm not super familiar on the ETH set up, just wanna make sure that we're only expecting it to be for these new coins

Reply:
I believe so, both `eth` & `polygon` extend from `abstractEthLikeNewCoins` which are the only chains we need to implement nfts for atm.